*** liszt.txt	Fri Aug 16 01:30:03 2024
--- liszt_pedal-fixed.txt	Fri Aug 16 02:55:02 2024
***************
*** 742,750 ****
  29081 On ch=1 n=64 v=60
  29084 On ch=1 n=68 v=0
  29120 On ch=1 n=39 v=85
  29151 On ch=1 n=39 v=0
- 29154 Par ch=1 c=64 v=63
- 29165 Par ch=1 c=64 v=127
  29178 On ch=1 n=64 v=0
  29185 On ch=1 n=58 v=95
  29197 On ch=1 n=55 v=76
--- 742,750 ----
  29081 On ch=1 n=64 v=60
  29084 On ch=1 n=68 v=0
  29120 On ch=1 n=39 v=85
+ 29134 Par ch=1 c=64 v=63
+ 29145 Par ch=1 c=64 v=127
  29151 On ch=1 n=39 v=0
  29178 On ch=1 n=64 v=0
  29185 On ch=1 n=58 v=95
  29197 On ch=1 n=55 v=76
***************
*** 764,775 ****
  29574 On ch=1 n=63 v=90
  29592 On ch=1 n=67 v=0
  29615 On ch=1 n=40 v=100
  29654 On ch=1 n=40 v=0
- 29674 Par ch=1 c=64 v=63
- 29682 Par ch=1 c=64 v=0
- 29692 Par ch=1 c=64 v=63
  29696 On ch=1 n=61 v=108
- 29706 Par ch=1 c=64 v=127
  29713 On ch=1 n=63 v=0
  29716 On ch=1 n=56 v=97
  29821 On ch=1 n=64 v=63
--- 764,775 ----
  29574 On ch=1 n=63 v=90
  29592 On ch=1 n=67 v=0
  29615 On ch=1 n=40 v=100
+ 29620 Par ch=1 c=64 v=63
+ 29628 Par ch=1 c=64 v=0
+ 29638 Par ch=1 c=64 v=63
+ 29652 Par ch=1 c=64 v=127
  29654 On ch=1 n=40 v=0
  29696 On ch=1 n=61 v=108
  29713 On ch=1 n=63 v=0
  29716 On ch=1 n=56 v=97
  29821 On ch=1 n=64 v=63
